lib/unitarize_links_quda.cu

```diff
@@ -335,12 +335,12 @@ namespace{
       for(int dir=0; dir<4; ++dir){
 	if(field.Precision() == QUDA_SINGLE_PRECISION){
 	  copyArrayToLink(&link, ((float*)(field.Gauge_p()) + (i*4 + dir)*18)); // order of arguments?
-	}else if(field.Precision() == QUDA_DOUBLE_PRECISION){     
+	}else if(field.Precision() == QUDA_DOUBLE_PRECISION){
 	  copyArrayToLink(&link, ((double*)(field.Gauge_p()) + (i*4 + dir)*18)); // order of arguments?
```

Comment:
Is it worth taking the time to turn this into an `operator=` workflow as opposed to a `load/store`-esque workflow like we've done elsewhere?

Reply:
This kernel has already been significantly rewritten in develop branch.  More cleanup certainly possible, but not in this pull.